src/main/java/movie/tickets/annotation/FieldsValueMatchValidator.java

```diff
@@ -0,0 +1,24 @@
+package movie.tickets.annotation;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldsValueMatchValidator implements ConstraintValidator<FieldsValueMatch, Object> {
```

Comment:
We can simplify the all method if we will use userregisterdto instead of object, then we can operate with fields of the dto and check passwords for equality.